javascript/tests/nw_tutorial/part1/duckduckgo_test_local.js

```diff
@@ -4,11 +4,13 @@ module.exports = {
     test: client => {
         client
             .url('https://duckduckgo.com/')
-            .waitForElementVisible('#content_homepage', 10 * 1000)
-            .setValue('#search_form_input_homepage', 'Nightwatch.js')
-            .setValue('#search_form_input_homepage', client.Keys.ENTER)
-            .assert.containsText('#r1-0 .result__title', 'Nightwatch.js')
-            .click('#r1-0 .result__title')
+            .waitForElementVisible('[data-testid=hero]', 10 * 1000)
+            .click('#searchbox_input')
+            .setValue('#searchbox_input', 'Nightwatch.js')
+            .setValue('#searchbox_input', client.Keys.ENTER)
```

Comment:
```suggestion
            .setValue('#searchbox_input', ['Nightwatch.js', client.Keys.ENTER])
```
need to update this as well so that it is equivalent to the Loadero example